src/components/app.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import url from 'url';
+
+import '../../node_modules/jquery/src/jquery';
 import '../../node_modules/uswds/dist/js/uswds.js';
+import '../../node_modules/bootstrap/dist/js/bootstrap.js';
+
 import '../../node_modules/@cmsgov/design-system-core/dist/index.css';
+import '../../node_modules/bootstrap/dist/css/bootstrap.css';
```

Comment:
Good question! Doesn't look like it -- removed them in 07ff302961da611d50249f4c276b9c3ce99c2281